src/logic/safe/store/models/types/gateway.d.ts

```diff
@@ -205,7 +205,7 @@ type MultiSigConfirmations = {
   signature: string | null
 }
 
-type TokenType = 'ERC721' | 'ERC20' | 'ETHER'
+export type TokenType = 'ERC721' | 'ERC20' | 'ETHER'
```

Comment:
`transaction.d.ts` also exports an enum `TokenType` with the same values. Maybe we could just use that everywhere?

Reply:
That's a good idea, but I would not do it in this PR, to avoid mixing things. wdyt?